dp_wizard/shiny/dataset_panel.py

```diff
@@ -9,7 +9,7 @@
     PUBLIC_PRIVATE_TEXT,
 )
 from dp_wizard.utils.csv_helper import get_csv_names_mismatch
-from dp_wizard.app.components.outputs import (
+from dp_wizard.shiny.components.outputs import (
```

Comment:
Add more welcome info on the dataset panel:
- "DP Wizard" title
- Relationship to the OpenDP project
- Link to github / pypi?

Reply:
> Add more welcome info on the dataset panel:
> 
> * "DP Wizard" title
> * Relationship to the OpenDP project
> * Link to github / pypi?

I don't see any changes in the dataset panel, are you planning on doing that in this PR?